http_handler_list_test.go

```diff
@@ -24,7 +26,19 @@ func (m NamespaceListerMock) ListNamespaces(ctx context.Context, username string
 }
 
 var _ = Describe("HttpHandlerList", func() {
-	const userHeader = "X-Email"
+	var request *http.Request
+
+	BeforeEach(func() {
+		var err error
+		ctx := context.WithValue(context.TODO(), namespacelister.ContextKeyUserDetails,
```

Comment:
Instead of context.TODO(), we can pull a context from ginkgo.  This means that any ginkgo cancellations will get passed along down here.

```suggestion
	BeforeEach(func(c context.Context) {
		var err error
		ctx := context.WithValue(c, namespacelister.ContextKeyUserDetails,
```

Reply:
done in 2620bcd4b09b4cac84ad16713c51ef97adb169ac